javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,18 @@
+function getFullname(firstName, surName, useFormalName, gender) {
+
+  if (useFormalName, gender === "male") {
+    return ("Lord" + " " + firstName + " " + surName);
```

Comment:
I gave to give space between the names (Lord Firstname Surname) otherwise it displays as LordFirstnameSurname. I am not sure is it the right way to add parenthesis for that.

Reply:
I misunderstood  first. Got it. Changed.